bin/rds-snapshot-export-pipeline-cdk.ts

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import { RdsSnapshotExportPipelineStack } from '../lib/rds-snapshot-export-pipeline-stack';
+import config = require('config');
```

Comment:
We should use the same way in another files @nbkhoi it should be imported: **import * as config from 'config'**

Reply:
fixed